chains/solana/contracts/programs/ccip-router/src/instructions/v1/onramp.rs

```diff
@@ -99,49 +57,22 @@ pub fn ccip_send<'info>(
         accounts_per_sent_token.push(current_token_accounts);
     }
 
-    let token_billing_config_accounts = accounts_per_sent_token
-        .iter()
-        .map(|accs| validated_try_to::billing_token_config(accs.fee_token_config, accs.mint.key()))
-        .collect::<Result<Vec<_>>>()?;
-
-    let per_chain_per_token_config_accounts = accounts_per_sent_token
-        .iter()
-        .map(|accs| {
-            validated_try_to::per_chain_per_token_config(
-                accs.token_billing_config,
-                accs.mint.key(),
-                dest_chain_selector,
-            )
-        })
-        .collect::<Result<Vec<_>>>()?;
-
-    let (fee, processed_extra_args) = fee_for_msg(
+    // Fee Quoter validates the message, calculates the fee in both the billing token
+    // and LINK, asserts that it doesn't exceed a maximum, and returns it
```

Comment:
Nit: Since this comment is just describing what the function does, maybe it would be better as a doc comment of `get_fee_cpi` instead.